docs/index.rst

```diff
@@ -0,0 +1,35 @@
+.. toctree::
+   For Enterprise <https://tidelift.com/subscription/pkg/pypi-|project|?utm_source=pypi-|project|&utm_medium=referral&utm_campaign=docs>
```

Comment:
I'm beginning to think this whole code block and the For Enterprise link for the TOCtree should be provided by substitutions or sphinx directives by way of an extension.

Reply:
I tried using a substitution for the TOCtree, but Sphinx didn't like that:

```
/Users/jaraco/code/public/pypa/setuptools/docs/index.rst:9:toctree contains reference to nonexisting document '|For Enterprise|'
```

Reply:
Oh, wait! Doesn't `toctree` require references to existing documents? I'm pretty sure that it's not designed to link arbitrary URLs. External links are often declared in some var in `conf.py` and are then rendered by the template in a designated location.

Reply:
For example, alabaster has a special var for external links map that get rendered right under the ToC (https://github.com/sanitizers/octomachinery/blob/5e39613/docs/conf.py#L219-L222) and furo has a var for arbitrary HTML (https://github.com/urllib3/urllib3/blob/4cf9a72/docs/conf.py#L77-L82).

Reply:
> Doesn't `toctree` require references to existing documents?

It also allows for explicit external references ([doc](https://www.sphinx-doc.org/en/1.5/markup/toctree.html#directive-toctree)), which is the feature that urllib3 uses.

> alabaster has a special var... furo has a [different] var

I'd like to avoid an implicit dependency on a theme. That's part of what makes the transition from alabaster to furo difficult - once the tidelift functionality depended on alabaster, switching to any other theme requires re-thinking the tidelift enrollment. And I definitely don't want to get into the business of supporting a bespoke solution for each and every project. I'd be okay with switching from alabaster to furo here, and then all of the tidelift projects downstream would adopt that constraint, but it would be even better if the concerns of tidelift sponsorship and theme selection could be kept independent.

Reply:
I'm starting to think that I'll just need to create a manual process for substituting the project name, given the weaknesses of rst and sphinx.

Reply:
The only way to make this theme-independent I can think of is by using JS. Also, if you were to have a dedicated RST document, maybe you could do that on jinja level by substituting the entire template but this would be totally theme-independent, more like a landing page.